### PR TITLE
Adding LazyRecord ORM

### PR DESCRIPTION
**Pull request by c9s**

### CODE REVIEW

**Comment by c9s** [comment]
Hi @ziadoz Could we merge this PR? Thanks  :)

**Comment by cosminordean** [comment]
wdzmarsdwndztmxdft mygnnm gftxf xfx

File: README.md [hunk 1]
```diff
@@ -432,6 +432,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Doctrine](http://www.doctrine-project.org/) - A comprehensive DBAL and ORM.
 * [Doctrine Extensions](https://github.com/l3pp4rd/DoctrineExtensions) - A collection of Doctrine behavioural extensions.
 * [Propel](http://www.propelorm.org/) - A fast ORM, migration library and query builder.
+* [LazyRecord](http://github.com/c9s/LazyRecord) - A fast ORM that builds your schema into static class files and offers you a smarter automatic migration mechanism.
```
Comment (jadb) [inline README.md]: Could this be shortened? The official repo's description is and nicer (no marketing buzz please). :wink:

Comment (c9s) [inline README.md]: Yes, sure! let me update this

**Comment by c9s** [comment]
Hi @jadb I just updated the patch, please take a look. Thanks!

**Comment by jadb** [comment]
@c9s sorry I should have better expressed myself. 

Please change to: A fast ORM designed for simplicity, extendability and performance. 

Wouldn't like usage of `The` and capitalization of `Fast` to sound like the *chosen* one ;)

**Comment by c9s** [comment]
Hi @jadb  Thanks for your help. the commit is updated. :)

**Comment by jadb** [comment]
Thanks!

**Pull request merged**